cpp/include/cudf_test/print_utilities.cuh

```diff
@@ -25,7 +25,7 @@
 
 #include <type_traits>
 
-namespace cudf::test::print {
+namespace cudf::test {
```

Comment:
These seem like unrelated changes. Are these necessary for this PR?

Reply:
Well, no, but it's a conflict right now if you include `cudf_test/column_utilities.hpp` and `cudf_test/print_utilities.hpp` because the former has a function called `cudf::test::print` and the latter has a namespace called `cudf::test::print`. Printing the array was a useful utility for me so I figured I should fix it

Reply:
Ouch. Can you separate that into another PR? I can quickly approve that one, but I want to keep the comparators work relatively clean because of its high complexity.